software/turtlebot4_simulator.md

```diff
@@ -22,30 +22,30 @@ Install useful development tools:
 sudo apt install ros-dev-tools
 ```
 
-### Ignition Gazebo
+### Gazebo
 
-{% tabs ignition %}
-{% tab ignition galactic %}
+{% tabs gazebo %}
+{% tab gazebo galactic %}
 
-Ignition Edifice must be installed:
+Gazebo Edifice must be installed:
 
 ```bash
 sudo apt-get update && sudo apt-get install wget
 sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
 wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
-sudo apt-get update && sudo apt-get install ignition-edifice
+sudo apt-get update && sudo apt-get install gz-edifice
```

Comment:
Can edifice be installed under the `gz` name? I thought it started from fortress.